server/api.js

```diff
@@ -77,10 +77,59 @@ async function getSensor(req, res) {
   }
 }
 
+const validateMessageClients = Validator.body(['braveKey', 'data']).exists()
```

Comment:
Yes! I was not sure how important the validators would be for the API calls, because there is also the `authorize` function that ensures that the given `braveKey` is valid... so I decided to _only_ make sure that these fields exist.

Because this API call is `POST`ed, I believe putting `braveKey` in the body of the request is better than in the query (which is what was done for API `GET` functions), and then `authorize` ensures that the braveKey is valid (handles the case where it is empty, too, which is invalid).

I can put a check in `messageClients` to return early in the case that `data` is empty.

Is this all reasonable?

Reply:
Thanks for explaining! That all seems reasonable to me. Thanks!